src/domain/consents/generateChallenge.ts

```diff
@@ -54,7 +54,7 @@ export function isConsentRequestInitiatedByValidSource (
   request: Request,
   consent: Consent): boolean {
   const fspiopSource = request.headers[Enum.Http.Headers.FSPIOP.SOURCE]
-  return (consent && consent.initiatorId === fspiopSource)
+  return (consent?.initiatorId === fspiopSource)
```

Comment:
What happens when this is compiled to JS?

Reply:
not sure but ask pawel?